models/evms/evms_blocks.sql

```diff
@@ -0,0 +1,143 @@
+{{ config(
+        alias ='blocks',
+        unique_key=['blockchain', 'number'],
+        post_hook='{{ expose_spells_hide_trino(\'["ethereum", "polygon", "bnb", "avalanche_c", "gnosis", "fantom", "optimism", "arbitrum"]\',
```

Comment:
is this intentionally hidden?

Reply:
same comment throughout all new spells here

Reply:
ah no good catch, will change to unhide them